app/views/registers/show.html.haml

```diff
@@ -1,5 +1,5 @@
-- @page_title = "#{@register.seo_title.present? ? @register.seo_title : @register.register_name + ' register'}"
-- @page_description = "#{@register.meta_description.present? ? @register.meta_description : @register.register_description}"
+- @page_title = "#{[@register.seo_title, @register.register_name].find(&:present?)}"
```

Comment:
do you still want to append `register` on the end? Or only for the `register_name` maybe?

Reply:
Yes, you are right.